requestmanager/asyncloader/unverifiedblockstore/unverifiedblockstore.go

```diff
@@ -30,21 +33,26 @@ func New(storer ipld.Storer) *UnverifiedBlockStore {
 // comes in as part of a traversal.
 func (ubs *UnverifiedBlockStore) AddUnverifiedBlock(lnk ipld.Link, data []byte) {
 	ubs.inMemoryBlocks[lnk] = data
+	ubs.dataSize = ubs.dataSize + uint64(len(data))
```

Comment:
I guess we don't need to synchronize because anyway the map is not synchronized, which suggests that this component is only ever called serially?

Reply:
@raulk Yeah - it's all called inside an event loop.